apiserver/facades/client/client/client.go

```diff
@@ -881,7 +881,7 @@ func (c *Client) toolVersionsForCAAS(args params.FindToolsParams, streamsVersion
 			}
 		}
 		arch, err := reg.GetArchitecture(imageName, number.String())
-		if errors.IsNotFound(err) {
+		if errors.IsNotFound(errors.Cause(err)) {
```

Comment:
What's the rationale for this - and maybe it should go in a separate PR?

Reply:
Yeah, this was for testing purposes. It will be in a separate PR with tests,